### ISSUE TITLE
alasql cli arguments don't seem to work on GitLabCI

### ISSUE THREAD

**Issue by oncletom** [agershun/alasql#1090]
> AlaSQL is based on unpaid voluntary work. Thank you for taking the time to make it better. 

Thanks so much for providing this tool. I'm enjoying this discovery.

Question about how to ...
- Please use http://stackoverflow.com/questions/ask?tags=alasql

Something is not working as expected:
- Describe the problem

When I run `alasql --file something.sql` on GitLabCI, I am welcomed with this message:

> No SQL to process

- Provide code that replicates the problem 

I created this repository to describe the problem: https://gitlab.com/oncletom/alasql-cli-arguments-issue, the command `npm test` fails (see [job output](https://gitlab.com/oncletom/alasql-cli-arguments-issue/-/jobs/154288915)).

If I clone this repo locally with `node@v10.15.0` and `npm@v6.5.0`, the same command `npm test` works.

If I do reuse the same image as on GitLabCI, with Docker, the same command `npm test` works.

```bash
docker run -ti --rm node:10 sh -c "git clone https://gitlab.com/oncletom/alasql-cli-arguments-issue.git && cd alasql-cli-arguments-issue && npm ci && npm test"
```

I have not been able to reproduce this problem elsewhere.

**Comment by mathiasrw** [comment]
I am glad its doing something good for you. 

I have a feeling its something about the working directory in the CI enviroment. The error comes when the files is empty or cant be found. If you do a `cat something.sql` do you get the right code?

**Comment by oncletom** [comment]
Yeah it's weird. I opened an issue here: https://gitlab.com/gitlab-org/gitlab-runner/issues/3950
